src/openstudio_app/main.cpp

```diff
@@ -215,7 +215,8 @@ int main(int argc, char* argv[]) {
     openstudio::OpenStudioApp app(argc, argv);
     openstudio::Application::instance().setApplication(&app);
 
-    app.setHighDpiScaleFactorRoundingPolicy(Qt::HighDpiScaleFactorRoundingPolicy::RoundPreferFloor);
+    // cf #535
+    app.setHighDpiScaleFactorRoundingPolicy(Qt::HighDpiScaleFactorRoundingPolicy::PassThrough);
```

Comment:
@macumber Do you remember why you had picked RoundPreferFloor (or actually, explicit set it, since PassThrough is the default)?
https://github.com/openstudiocoalition/OpenStudioApplication/blame/f3d7afb29064b40d75ac8c3c7ec6ca02d48b18b3/src/openstudio_app/main.cpp#L218

Reply:
I remember experimenting with the different options and thinking that one was the best but whatever reason wouldn't outweigh the offset issue